StartupSession/Link/Config.apln

```diff
@@ -194,7 +194,9 @@
      
           :Trap 0
               config←0 ⎕JSON⍠('Hierarchy' 0)('Dialect' 'JSON5')⊢json←⊃⎕NGET file 0
-              ⍝⍝ /// reconstitute matrix settings
+              :If 2=⎕NC 'config.Settings.typeExtensions'
+                  config.Settings.(typeExtensions←↑typeExtensions)
```

Comment:
Don't just blindly mix. Check if actually a split matrix. Also, if just `(number charvec/scal)` then add a leading axis. When done, do a sanity check that we have exactly 2 columsn, the left being entirely numeric and the right being only charvecs and charscals.